libs/sdk-core/src/breez_services.rs

```diff
@@ -1586,16 +1597,31 @@ impl BreezServices {
             .register_payment_notifications(
                 lsp_info.id,
                 lsp_info.lsp_pubkey,
-                webhook_url,
+                webhook_url.clone(),
                 sign_response.signature,
             )
             .await?;
 
+        for swap in self
+            .btc_receive_swapper
+            .list_monitored()?
+            .iter()
+            .filter(|swap| !swap.refundable())
+        {
+            let swap_address = &swap.bitcoin_address;
+            info!("Found non-refundable monitored swap with address {swap_address}, registering for swap tx notifications");
+            self.register_swap_tx_notification(swap_address).await?;
```

Comment:
This will register now with the old url as it is not saved in the db.

Reply:
Fixed: 6eb54b06f4ddefbec9066085d7ee457095b97431